test/verify/machineslib.py

```diff
@@ -1404,6 +1409,29 @@ def installWithErrorTest(dialog):
                                       {"Operating System": "You need to select the most closely matching Operating System"})
 
         # try to CREATE few machines
+        virtInstallVersion = self.machine.execute("virt-install --version")
+        if virtInstallVersion >= "2.2.0":
```

Comment:
This was not addressed. Once any of our bots disctros get's version of virt-install 2.10, 2.11..., it will begin to fail the test.
But that might not happen, and if that happens, we can fix it then.

Reply:
Done